libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1669,17 +1674,30 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 			<?isECRecover>
 				mstore(0, 0)
 			</isECRecover>
-			let <success> := <call>(<gas>, <address> <?isCall>, 0</isCall>, <pos>, sub(<end>, <pos>), 0, 32)
+			<?eof>
+				// EOF always uses extstaticcall
```

Comment:
Why?

Reply:
regular `call` for for this case was used only for rev before `staticcall` was introduced (before Byzantium). For EOF we have `extstaticcall` already.